examples/mean-reversion.go

```diff
@@ -0,0 +1,255 @@
+package main
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/alpacahq/alpaca-trade-api-go/alpaca"
+	"github.com/alpacahq/alpaca-trade-api-go/common"
+	"github.com/shopspring/decimal"
+)
+
+type alpacaClientContainer struct {
+	client         *alpaca.Client
+	closingPrices  []float64
+	runningAverage float64
+	lastOrder      string
+	timeToClose    float64
+	stock          string
+}
+
+var alpacaClient alpacaClientContainer
+
+func init() {
+	API_KEY := "API_KEY"
```

Comment:
I generally like to have these as like a "replaceme" or "your api key here" value just so it's clearer to the users where their stuff goes.